### PR TITLE
OSDOCS-6285: Removed VMC doc set and references from openshift repo

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-6285](https://issues.redhat.com/browse/OSDOCS-6285)

Version(s):
4.14

Link to docs preview:
[Link to vSphere doc set becasue VMC doc set is removed](https://60614--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/preparing-to-install-on-vsphere.html#installation-vsphere-infrastructure_preparing-to-install-on-vsphere)

^^ Note that the previous link points to the statement "You can host the VMware vSphere infrastructure on-premise or on a VMware Cloud Verified provider that meets the requirements outlined in the following table".

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: modules/installation-initializing.adoc [hunk 1]
```diff
@@ -219,9 +205,6 @@ endif::ibm-cloud[]
 ifdef::osp[]
 {rh-openstack-first}.
 endif::osp[]
-ifdef::vsphere,vmc[]
-VMware vSphere.
-endif::vsphere,vmc[]
```
Comment (bscott-rh) [inline modules/installation-initializing.adoc:224]: Did you mean to only delete the `vmc` parameter here?

File: modules/installation-obtaining-installer.adoc
Comment (bscott-rh) [inline modules/installation-obtaining-installer.adoc:1]: Make sure to remove the un-setting of the vmc attributes at the bottom of this file.

File: modules/installation-initializing.adoc (hunk 1 above)
Comment (dfitzmau) [reply modules/installation-initializing.adoc:224]: Thanks for the review, @bscott-rh . Yes I did. Do you know of a reason as why I should keep this parameter?

Comment (dfitzmau) [reply modules/installation-initializing.adoc]: Thanks for the review, @bscott-rh . Well spotted. I corrected re-added back in

```
ifdef::vsphere[]
VMware vSphere.
endif::vsphere[] 
```

File: modules/installation-obtaining-installer.adoc
Comment (dfitzmau) [reply modules/installation-obtaining-installer.adoc:1]: Sorry, I'm unsure of this requests. The VMC instances were removed. However, I did notice that the vSphere attributes were not unset, so I fixed this for this file.

File: modules/installation-vsphere-infrastructure.adoc [hunk 2]
```diff
@@ -75,39 +41,9 @@ You can host the VMware vSphere infrastructure on-premise or on a link:https://c
 |vSphere 7.0 Update 2 and later
 |This plugin creates vSphere storage by using the in-tree storage drivers for vSphere included in {product-title}.
 
-ifndef::vmc[]
-|Optional: Networking (NSX-T)
-|vSphere 7.0 Update 2 and later
-|vSphere 7.0 Update 2 is required for {product-title}. For more information about the compatibility of NSX and {product-title}, see the Release Notes section of VMware's link:https://docs.vmware.com/en/VMware-NSX-Container-Plugin/index.html[NSX container plugin documentation].
-endif::vmc[]
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply modules/installation-vsphere-infrastructure.adoc]: Well spotted fixed.

File: modules/installation-obtaining-installer.adoc
Comment (bscott-rh) [reply modules/installation-obtaining-installer.adoc:1]: Interesting find... looks like the :vmc: AND :vsphere: variables were never unset at the end of this file.

**Pull request merged**